### PR TITLE
Add postman collection for RNC app.

### PR DESCRIPTION
**Pull request by marekzatko**
Signed-off-by: marekzatko <Marek.Zatko@pantheon.tech>

### CODE REVIEW

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+					"response": []
+				},
+				{
+					"name": "Get NETCONF device status",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": ""
+						}
```
Comment (martinbugan-pt) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:69]: empty URL - fix this

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+			"name": "Device data manipulation",
+			"item": [
+				{
+					"name": "GET NETCONF device data",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": ""
+						}
```
Comment (martinbugan-pt) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:111]: empty URL - fix this

**Review by martinbugan-pt — CHANGES_REQUESTED** [review]

**Review by martinbugan-pt — APPROVED** [review]

**Review by martinbugan-pt — APPROVED** [review]

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+								"yang-ext:mount",
+								"toaster:toaster"
+							]
+						},
+						"description": "Example request for setting configuration data on NETCONF device. This request is valid if the device does implement toaster yang."
+					},
+					"response": []
+				},
+				{
+					"name": "GET restconf operations",
```
Comment (Pyro2266) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:186]: This is not a "Device data manipulation"

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Device managment",
+			"item": [
+				{
+					"name": "Connect NETCONF device",
+					"request": {
+						"auth": {
```
Comment (Pyro2266) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:15]: This is the only request with basic-auth. Either add it to all requests or remove it.

File: lighty-applications/lighty-rnc-app-aggregator/README.md [hunk 5]
```diff
@@ -122,7 +122,9 @@ To build and start the RNC lighty.io application using docker in the local envir
 
 5. Test the RNC lighty.io application. Default RESTCONF port is `8888`
 
-
+## Example Postman collection
+For convenience, we provide [postman-collection](lighty-rnc-app/Lighty-RNC.postman_collection.json) which contains
+ example http requests (RESTCONF and Akka) for communicating with Lighty RNC application.
```
Comment (Pyro2266) [inline lighty-applications/lighty-rnc-app-aggregator/README.md:127]: Add a note that IPs/ports used in requests may differ depending on deployment

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+								""
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Cluster managment",
```
Comment (Pyro2266) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:212]: Does this "direct access" require some special k8s (microk8s) configuration?

Comment (Pyro2266) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:212]: Add example requests for node-port and ingress too (get/put data should be enough)

**Review by Pyro2266 — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json (hunk 3 above)
Comment (marekzatko) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json]: Done, removed the request, it is very similar to others.

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json (hunk 4 above)
Comment (marekzatko) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json]: Done, removed it

File: lighty-applications/lighty-rnc-app-aggregator/README.md (hunk 5 above)
Comment (marekzatko) [reply lighty-applications/lighty-rnc-app-aggregator/README.md]: Done

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json (hunk 6 above)
Comment (marekzatko) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app/Lighty-RNC.postman_collection.json:173]: 1. GET cluster/members does not require anything special. The down/leave member may require akka config setting, it is mentioned in description of the request.
2. Done, added

**Pull request merged**